pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/DownloadParticipantListRoute.java

```diff
@@ -0,0 +1,181 @@
+                    } else {
+                        esAlias = Alias.of(column.getTableAlias());
+                    }
+                    return esAlias.getValue().isEmpty()? column.getName() : esAlias.getValue() + "." + column.getName();
+                })
+                .collect(Collectors.toList());
+
+        Filterable filterable = FilterFactory.of(request);
+        ParticipantWrapperResult filteredList = (ParticipantWrapperResult) filterable.filter(request.queryMap());
+        Workbook workbook = new XSSFWorkbook();
+        List<String> headerNames = payload.getHeaderNames();
+        Sheet sheet = workbook.createSheet("Participant List");
+        Row header = sheet.createRow(0);
+        IntStream.range(0, headerNames.size()).forEach(i -> {
+            Cell cell = header.createCell(i);
+            cell.setCellValue(headerNames.get(i));
+            sheet.autoSizeColumn(i);
+        });
+        int currentRow = 1;
+        for (ParticipantWrapperDto participant : filteredList.getParticipants()) {
+            List<Object> excelRow = new ArrayList<>();
+            Map<String, Object> esDataAsMap = participant.getEsDataAsMap();
+            List<String> row = new ArrayList<>();
+            for (String esAlias : esAliases) {
+                Object nestedValue = getNestedValue(esAlias, esDataAsMap);
+                excelRow.add(nestedValue);
+                if (nestedValue instanceof Collection) {
+                    Collection<?> value = (Collection<?>) nestedValue;
+                    row.add(value.stream().map(Object::toString)
+                            .collect(Collectors.joining(System.lineSeparator())));
+                } else {
+                    row.add(nestedValue.toString());
+                }
+            }
+            currentRow = createRecord(sheet, currentRow, new ParticipantRecord(excelRow));
+        }
+        responseFile(response, workbook);
+        return response.raw();
+    }
+
+
+    private void responseFile(Response response, Workbook workbook) throws Exception {
+        File currDir = new File(".");
+        String path = currDir.getAbsolutePath();
+        String strDate = getFormattedDate();
+        String fileLocation = String.format("%sParticipant-%s.xlsx", path.substring(0, path.length() - 1), strDate);
+        FileOutputStream outputStream = new FileOutputStream(fileLocation);
+        workbook.write(outputStream);
+        workbook.close();
+        File file = new File(fileLocation);
+        response.raw().setContentType("application/octet-stream");
+        response.raw().setHeader("Content-Disposition", "attachment; filename=" + file.getName());
+        byte[] encoded = Files.readAllBytes(Paths.get(fileLocation));
+        ServletOutputStream os = response.raw().getOutputStream();
+        os.write(encoded);
+        os.close();
+        Files.deleteIfExists(file.toPath());
+    }
+
+    private String getFormattedDate() {
+        Date date = new Date();
```

Comment:
`Date` is outdated I would say, please change it to `LocalDate` and its relevant formatter.